unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/ChatController.cs

```diff
@@ -143,36 +143,45 @@ public void UpdateChannelMembers(string payload)
     [UsedImplicitly]
     public void UpdateChannelInfo(string payload)
     {
-        var msg = JsonUtility.FromJson<ChannelInfoPayload>(payload);
-        var channelId = msg.channelId;
-        var channel = new Channel(channelId, msg.unseenMessages, msg.memberCount, msg.joined, msg.muted, msg.description);
-        var justLeft = false;
+        var msg = JsonUtility.FromJson<ChannelInfoPayloads>(payload);
 
-        if (channels.ContainsKey(channelId))
+        foreach (var channelInfo in msg.channelInfoPayload)
         {
-            justLeft = channels[channelId].Joined && !channel.Joined;
-            channels[channelId].CopyFrom(channel);
-        }
-        else
-            channels[channelId] = channel;
+            var channelId = channelInfo.channelId;
+            var channel = new Channel(channelId, channelInfo.unseenMessages, channelInfo.memberCount, channelInfo.joined, channelInfo.muted, channelInfo.description);
+            var justLeft = false;
 
-        if (justLeft)
-        {
-            OnChannelLeft?.Invoke(channelId);
+            if (channels.ContainsKey(channelId))
+            {
+                justLeft = channels[channelId].Joined && !channel.Joined;
+                channels[channelId].CopyFrom(channel);
+            }
+            else
+                channels[channelId] = channel;
 
-            // TODO (responsibility issues): extract to another class
-            AudioScriptableObjects.leaveChannel.Play(true);
-        }
+            if (justLeft)
+            {
+                OnChannelLeft?.Invoke(channelId);
 
-        OnChannelUpdated?.Invoke(channel);
+                // TODO (responsibility issues): extract to another class
+                AudioScriptableObjects.leaveChannel.Play(true);
```

Comment:
careful with this one, we may play this audio many times in a row.. it should be better if you pull it out of the loop

Reply:
Fixed on https://github.com/decentraland/unity-renderer/pull/3145/commits/d8bbde8eb5cadb94e726d632de08116c189cfaeb